changelogs/unreleased/00002-dymurray.md

```diff
@@ -0,0 +1 @@
+Pull in proxy configuration from environment when constructing custom transport
```

Comment:
let's go with the same file name convention as for core velero - so `37-dymurray`

Reply:
Sorry about that... not enough coffee. Saw 00001 and said "oh I guess I'm 00002..."

Fixed!